src/index.js

```diff
@@ -4,8 +4,8 @@ import cx from 'classnames';
 import debounce from './lib/debounce';
 
 class OnVisible extends Component {
-    constructor() {
-        super(...arguments);
+    constructor(props) {
+        super(props);
```

Comment:
Is this actually the case? In the [React documentation](https://facebook.github.io/react/docs/reusable-components.html#setting-the-initial-state) they seem to only invoke `super` with the props argument. Curious as to why they'd do that if what you say is true